device_parser.c

```diff
@@ -49,7 +48,7 @@ static void nextsym(struct device_parser *dp)
 	}
 }
 
-static int accept(struct device_parser *dp, int type, char *scalar)
+int device_parser_accept(struct device_parser *dp, int type, char *scalar)
```

Comment:
Please pass length as an argument. I feel bad about exporting TOKEN_LENGTH in the header.

Reply:
Done